src/lib/Controller.js

```diff
@@ -90,16 +91,15 @@ export default class Controller {
       })
     })
 
-    // migrate from extension storage to indexedDB
+    // migrate from localForage back to extension storage
 
-    browser.storage.local
-      .get('accounts')
+    localForage.getItem('accounts')
       .then(async d => {
         if (!d.accounts) return
         return AccountStorage.changeEntry('accounts', () => d.accounts)
```

Comment:
Gnaaa, this doesn't work. I should have noticed.

Reply:
hm, I'm sorry to hear that. I tested the upgrade process using the version of the `develop` branch and had success getting my accounts to update. 

I guess its a problem if you are coming from an older version that had the browser storage to begin with.